keras_nlp/models/roberta.py

```diff
@@ -170,6 +171,184 @@ def get_config(self):
         return config
 
 
+class RobertaMultiSegmentPacker(keras.layers.Layer):
+    """Packs multiple sequences into a single fixed width model input.
+
+    This layer packs multiple input sequences into a single fixed width sequence
+    containing start and end delimiters, forming a dense input suitable for a
+    classification task for RoBERTa.
+
+    TODO(abheesht17): This is a temporary, unexported layer until we find a way
```

Comment:
Since we aren't exporting this, we can can probably cut the docstring way down and just leave the TODO above as a comment.